collections/_posts/2024-12-02-SSL_comparison.md

```diff
@@ -0,0 +1,123 @@
+---
+layout: review
+title: "Systematic comparison of semi-supervised and self-supervised learning for medical image classification"
+tags: classification, semi-supervision, self-supervision
+author: "Juliette Moreau"
+cite:
+    authors: "Zhe Huang, Ruije Jiang, Shuchin Aeron and Michael C. Hughes"
+    title:   "Systematic comparison of semi-supervised and self-supervised learning for medical image classification"
+    venue:   "CVPR, 2024"
+pdf: "https://arxiv.org/pdf/2307.08919"
+---
+
+
+# Highlights
+
+* Benchmark of SSL methods on medical images.
+* Implementation of a real life hyperparameters optimization.
+* Comparison of semi-supervised and self-supervised learning.
+
+# Note
+
+Code is available here: https://github.com/tufts-ml/SSL-vs-SSL-benchmark
+
+# Introduction
+
+In medical domain there is often a lack of annotated data for deep learning model training while there is a lot of unlabeled data in data health records. Semi-supervised and self-supervised learning are methods developed to take advantage of these unlabeled images to improve classification accuracy. The first one train classifiers jointly with two loss terms, while the second is a two-stages approach, the first to learn deep representation and the second to fine-tune the classifier. However, the two methods are rarely compared. The first question they try to answer is: **Which recent semi- or self-supervised methods are likely to be most effective?** \
+But those methods are very sensitive to hyperparameters, and benchmarks often don't consider this (either no hyperparameters tuning or tuning with a huge labeled validation set, bigger than the training set, which is not realistic). In this paper, they try to take that in account by answering the question: **Given limited available labeled data and limited compute, is hyperparameter tuning worthwhile?**
```

Comment:
benchmarks often don't consider **this** : Not very clear imo, it could be more explicit. Maybe you could add another sentence instead of the note in parentheses